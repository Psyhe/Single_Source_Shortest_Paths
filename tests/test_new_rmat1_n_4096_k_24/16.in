4096 2736 2905
2736 26 86
2736 164 203
2736 3584 109
2736 64 112
2737 3140 4
2738 16 41
2738 2696 249
2739 72 206
2739 128 216
2740 2048 128
2740 1028 166
2740 64 87
2740 102 217
2741 2048 161
2741 209 145
2742 16 204
2744 132 206
2744 1180 148
2744 144 218
2745 64 203
2749 128 79
2752 3072 168
2752 1320 51
2752 2056 54
2752 714 99
2752 256 197
2752 1220 93
2752 8 34
2752 260 154
2752 2432 95
2752 12 215
2752 2362 76
2752 0 246
2752 16 153
2752 22 137
2752 1024 45
2752 1 45
2752 2308 174
2753 910 160
2753 768 49
2753 2884 152
2753 1028 168
2753 35 178
2753 66 196
2754 2048 12
2754 128 191
2754 1164 128
2755 39 247
2755 416 4
2755 36 224
2755 2320 12
2755 128 50
2756 264 86
2756 3338 82
2756 48 158
2756 1118 23
2756 10 142
2756 518 189
2757 0 122
2757 2304 119
2757 66 154
2758 2048 65
2758 740 25
2760 82 191
2760 2096 100
2760 1096 135
2760 1024 127
2760 544 90
2760 1024 229
2760 97 146
2760 388 237
2760 0 128
2760 8 140
2760 1728 30
2761 1281 48
2761 1026 107
2761 662 223
2762 144 235
2762 1809 162
2762 290 135
2763 24 70
2764 262 236
2764 1088 218
2764 132 41
2768 2050 178
2768 34 247
2768 1166 85
2768 264 246
2768 2176 172
2768 1347 18
2768 0 250
2768 328 131
2768 8 199
2769 160 44
2769 2822 183
2770 128 115
2771 1056 10
2771 1036 30
2772 200 67
2774 52 242
2774 528 100
2776 527 11
2776 2337 64
2777 513 251
2777 1632 25
2780 1632 200
2780 32 33
2784 12 174
2784 2080 36
2784 37 219
2784 1 148
2784 3104 135
2784 0 121
2784 0 81
2784 176 35
2785 0 180
2785 2052 137
2786 420 225
2786 4 88
2786 1880 191
2787 24 6
2788 395 221
2789 294 229
2790 2424 131
2792 1568 82
2792 1280 24
2792 2828 144
2792 64 202
2793 196 79
2794 432 218
2794 2605 118
2796 54 203
2796 264 68
2800 1224 78
2804 1686 124
2805 3968 167
2814 364 225
2816 2566 176
2816 1168 150
2816 346 44
2816 228 177
2816 1152 243
2816 12 72
2816 352 250
2816 2088 242
2816 654 214
2816 3752 19
2816 1040 177
2816 385 182
2816 8 208
2816 3596 132
2816 2080 158
2816 1088 15
2816 1092 245
2816 257 38
2816 1197 72
2816 145 122
2816 2129 58
2816 184 111
2816 547 110
2816 76 78
2816 2104 97
2816 144 159
2816 1157 80
2816 590 39
2816 322 37
2816 2576 109
2816 160 252
2816 136 20
2816 514 0
2816 1328 163
2816 785 247
2816 170 35
2816 24 23
2816 2448 205
2816 2093 157
2816 3072 253
2816 0 149
2816 0 19
2816 656 2
2816 3888 205
2816 512 11
2816 3232 202
2816 1416 76
2816 530 176
2816 36 46
2816 2064 175
2816 2008 163
2816 320 159
2816 0 182
2816 2662 128
2816 613 252
2816 1304 130
2816 15 210
2816 2176 199
2816 144 222
2816 162 47
2816 768 158
2816 288 190
2816 136 123
2816 2208 150
2816 1 146
2816 66 250
2816 449 23
2816 179 56
2816 19 18
2816 16 204
2816 192 60
2816 128 51
2816 2068 65
2816 512 122
2816 1280 49
2816 3210 180
2816 122 104
2816 1538 201
2816 1730 184
2816 608 214
2816 141 240
2816 102 21
2816 3120 56
2816 2 87
2816 2048 13
2816 12 51
2816 16 123
2816 1305 27
2816 384 23
2817 56 72
2817 326 177
2817 1862 218
2817 1988 182
2817 536 209
2817 8 120
2817 2058 225
2817 1 35
2817 2128 70
2817 260 233
2817 2304 123
2817 64 252
2817 130 27
2817 650 47
2817 1568 14
2817 782 44
2817 650 64
2817 1056 140
2817 2709 138
2817 2560 100
2817 0 155
2817 1332 16
2817 341 82
2818 1345 89
2818 2112 72
2818 195 103
2818 720 115
2818 1060 244
2818 34 15
2818 1089 121
2818 53 139
2818 66 57
2818 1049 33
2818 1160 183
2818 524 192
2818 424 140
2818 257 141
2818 4 8
2818 516 2
2818 1306 46
2818 1664 79
2818 3238 20
2818 392 107
2818 2022 190
2818 3104 112
2818 2216 162
2818 3713 111
2819 0 2
2819 1024 222
2819 1313 93
2819 1024 7
2819 2048 60
2819 320 167
2819 1072 114
2819 32 16
2819 328 131
2819 200 56
2819 2097 133
2820 0 49
2820 642 73
2820 1 29
2820 579 186
2820 3104 178
2820 165 62
2820 512 160
2820 1568 56
2820 252 251
2820 2066 216
2820 2338 180
2820 0 27
2820 1073 106
2820 2048 125
2820 12 200
2820 2562 90
2820 256 251
2820 128 103
2820 1185 3
2820 661 62
2820 4 245
2820 280 228
2820 2120 92
2821 2064 107
2821 520 98
2821 1536 250
2821 514 65
2821 1024 7
2821 64 127
2821 8 3
2822 2 176
2822 2052 227
2822 1 220
2822 515 165
2822 2769 183
2822 512 100
2822 576 228
2822 64 171
2823 536 81
2824 2380 53
2824 3616 60
2824 16 102
2824 73 217
2824 1548 232
2824 2457 0
2824 32 191
2824 256 171
2824 1250 100
2824 2048 161
2824 1 155
2824 784 22
2824 1301 155
2824 1024 62
2824 2135 255
2824 512 30
2824 2142 101
2824 51 60
2824 192 249
2824 1284 42
2825 672 177
2825 2560 228
2825 260 38
2825 1300 35
2826 80 107
2826 2083 80
2826 2313 221
2826 2465 217
2826 0 102
2826 48 111
2826 550 139
2826 1536 26
2826 1154 115
2826 184 195
2826 2840 163
2826 3074 87
2827 48 0
2827 3291 190
2828 128 240
2828 224 131
2828 2048 114
2828 8 157
2828 2792 144
2828 1 12
2828 3888 231
2828 1328 239
2829 1176 38
2829 547 6
2830 2568 98
2831 288 17
2832 1028 58
2832 1640 30
2832 96 232
2832 2309 15
2832 1209 105
2832 1153 75
2832 69 93
2832 2 4
2832 50 92
2832 1824 48
2832 2592 81
2832 2560 116
2832 3206 86
2832 16 139
2832 172 205
2832 1041 68
2832 1072 137
2833 1069 253
2833 132 47
2833 112 109
2833 258 134
2833 5 253
2833 576 236
2833 130 214
2833 1088 159
2833 3204 185
2833 2114 88
2833 1594 162
2833 8 199
2834 456 230
2834 1536 220
2834 512 86
2835 529 37
2836 1073 3
2836 10 55
2836 268 57
2836 3600 176
2836 64 211
2836 2353 99
2836 162 185
2836 1072 73
2836 523 189
2836 1589 29
2836 12 112
2836 288 12
2836 1058 148
2836 226 45
2836 2099 94
2837 2080 193
2837 1155 164
2837 42 206
2837 1066 253
2837 36 123
2837 9 187
2837 512 12
2838 2048 181
2838 17 202
2838 1089 127
2839 1297 245
2839 688 161
2840 20 193
2840 0 74
2840 2826 163
2840 34 154
2840 3176 190
2840 1306 22
2840 32 191
2842 32 240
2842 513 90
2842 16 193
2843 128 121
2844 225 57
2847 96 51
2848 640 183
2848 54 23
2848 3076 105
2848 260 184
2848 1036 9
2848 8 112
2848 1860 182
2848 216 161
2848 2068 121
2848 114 63
2848 1029 128
2848 516 35
2848 523 2
2848 16 1
2848 1152 121
2848 1088 87
2848 2050 251
2848 20 27
2848 1390 105
2848 524 223
2848 258 141
2848 656 110
2848 21 135
2848 3393 58
2849 272 52
2849 1542 56
2849 2084 109
2849 1064 0
2849 2281 120
2849 3222 89
2849 1044 138
2849 3214 163
2849 2048 60
2849 1156 148
2850 2112 9
2850 9 60
2850 652 89
2850 128 51
2850 2304 161
2850 1042 104
2850 1672 20
2851 32 96
2852 1318 233
2852 66 126
2852 128 187
2852 167 114
2852 5 1
2852 256 15
2852 112 21
2852 805 178
2852 1155 227
2852 132 237
2852 257 9
2853 62 200
2853 403 35
2854 1168 113
2854 1152 22
2854 1825 197
2856 1024 188
2856 291 195
2856 2082 167
2856 576 194
2856 128 127
2856 533 169
2856 452 200
2856 2560 124
2856 2241 180
2856 2304 222
2857 24 239
2860 0 207
2864 10 178
2864 1543 190
2864 1440 186
2864 10 61
2864 144 239
2864 836 41
2864 793 166
2864 0 59
2864 1540 139
2865 16 141
2865 3 111
2868 515 179
2868 513 252
2872 2116 10
2873 384 245
2874 457 245
2876 16 163
2880 681 79
2880 144 124
2880 660 212
2880 280 28
2880 1200 175
2880 512 175
2880 16 82
2880 200 80
2880 2056 108
2880 1025 152
2880 272 103
2880 16 87
2880 522 124
2880 272 197
2880 2700 142
2880 1040 104
2880 296 206
2880 1545 56
2880 3240 7
2880 1 217
2880 149 101
2881 1097 22
2881 1464 97
2881 2209 71
2881 272 95
2881 288 150
2882 72 82
2882 520 61
2882 1 126
2882 640 23
2882 393 253
2883 181 9
2883 1098 202
2883 792 156
2883 128 239
2884 2080 12
2884 579 94
2884 1068 96
2884 1040 180
2884 320 168
2884 2753 152
2885 32 187
2885 424 121
2885 2433 62
2886 2176 84
2886 64 159
2887 67 130
2887 2313 72
2888 32 79
2888 128 27
2888 3109 231
2888 40 71
2888 402 46
2888 0 169
2888 1024 39
2888 256 7
2888 672 215
2888 1 63
2889 8 180
2889 325 15
2890 1041 128
2892 1232 28
2892 8 106
2892 497 132
2892 0 80
2894 8 129
2896 3076 20
2896 1045 112
2896 1346 152
2896 1483 127
2896 614 33
2896 3075 178
2896 1601 145
2896 3216 124
2896 1189 73
2896 44 75
2896 4 123
2896 1118 126
2897 2048 247
2897 2572 46
2897 1 222
2898 1092 23
2898 1024 61
2898 176 65
2898 2056 195
2898 1200 121
2898 76 79
2900 1201 110
2900 1 120
2900 512 82
2900 83 8
2901 16 123
2901 1 99
2904 580 249
2904 8 199
2904 1024 89
2904 268 50
