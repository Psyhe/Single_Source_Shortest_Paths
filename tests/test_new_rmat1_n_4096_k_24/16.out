98
13
43
208
88
156
206
inf
206
204
inf
inf
inf
82
inf
inf
36
50
13
11
89
121
41
inf
53
58
140
72
43
inf
inf
inf
97
46
118
14
74
inf
103
inf
68
37
inf
inf
34
inf
inf
inf
41
139
89
8
223
255
inf
inf
30
89
259
inf
71
inf
inf
inf
84
inf
inf
inf
178
202
inf
inf
inf
inf
inf
inf
inf
inf
245
inf
7
30
4
2
27
5
104
83
30
38
28
5
13
21
101
18
5
49
90
40
13
16
135
180
48
inf
91
124
80
inf
inf
53
3
2
10
97
3
88
27
inf
130
241
inf
inf
207
inf
inf
inf
52
115
inf
inf
181
inf
inf
inf
22
246
268
inf
165
inf
inf
inf
16
28
26
23
17
63
85
97
8
20
131
inf
37
inf
131
inf
21
51
29
inf
27
100
inf
inf
69
inf
